src/ServerTelemetryChannel/SamplingTelemetryProcessor.cs

```diff
@@ -58,6 +62,16 @@ public SamplingTelemetryProcessor(ITelemetryProcessor next)
             };
         }
 
+        internal SamplingTelemetryProcessor(ITelemetryProcessor unsampledNext, ITelemetryProcessor sampledNext) : this(sampledNext)
```

Comment:
[not blocking] If we need it ourselves - does it mean that we should expose it for customers as well?

Reply:
Not certain anyone else uses it in the way we have with `AdaptiveSamplingTelemetryProcessor` (see below for a flow chart).